File: eform-client/src/app/models/common/pagination-model.ts
export class PaginationModel {
  pageIndex: number = 1;
  pageSize: 10;
  offset: 0;

  constructor(pageIndex?, pageSize?, offset?) {
    this.pageSize = pageSize;
    this.pageIndex = pageIndex;
    this.offset = offset;
  }
}

File: eFormAPI/eFromAPI.Common/Models/PaginationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eFromAPI.Common.Models
{
    public class PaginationModel
    {
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public int Offset { get; set; }
    }
}
